Services/data/devicemodels/scripts/engine-02-state.js

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+/*global log*/
+/*global updateState*/
+/*global updateProperty*/
+/*global sleep*/
+/*jslint node: true*/
+
+"use strict";
+
+// Default state
+var state = {
+    online: true,
+    fuellevel: 0.0,
+    fuellevel_unit: "Gal",
+    coolant: 7500.0,
+    coolant_unit: "ohm",
+    vibration: 10.0,
+    vibration_unit: "mm"
+};
+
+// Default device properties
+var properties = {};
+
+/**
+ * Restore the global state using data from the previous iteration.
+ *
+ * @param previousState device state from the previous iteration
+ * @param previousProperties device properties from the previous iteration
+ */
+function restoreSimulation(previousState, previousProperties) {
```

Comment:
same for this, move to common util ?

Reply:
same comment https://msazure.visualstudio.com/One/_workitems/edit/2304578